Polygon/eth_blockNumber.yaml

```diff
@@ -31,11 +31,4 @@ paths:
               const alchemy = new Alchemy(settings);
 
               // Get the latest block number
-              alchemy.core.getBlockNumber().then(console.log);
-      responses:
-          '200':
-            description: Returns integer of the current block number the client is on.
-            content:
-              application/json:
-                schema:
-                  $ref: ../evm_responses.yaml#/eth_blockNumber
```

Comment:
@opman129  does the response description still appear even without these?

Reply:
Yes it does. Comes directly from the $ref